NuGet.Config

```diff
@@ -9,6 +9,7 @@
     <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
     <add key="vside_vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="vside_vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vssdk-unifiedPIAs" value="https://devdiv.pkgs.visualstudio.com/DevDiv/_packaging/vssdk-unifiedPIAs/nuget/v3/index.json" />
```

Comment:
Is this a private feed? 
Packages need to be on public feeds for us to be able to consume them. The CI will likely fail.
Given that this is temp, there are 2 options I can think of:

* Change the CI so that auth works (NuGetAuthenticate task might be enough)
* Have them publish the package on a public feed. Req long term.